context/src/main/java/io/micronaut/runtime/server/EmbeddedServer.java

```diff
@@ -49,6 +49,11 @@ public interface EmbeddedServer extends EmbeddedApplication<EmbeddedServer> {
      */
     URL getURL();
 
+    /**
+     * @return The full URL to the server with the context path
+     */
+    URL getURLWithContextPath();
```

Comment:
introducing a new abstraction method in a patch release is a breaking change. Rename this method to `getContextURI()`, you should return URI instead of URL and make it a default method that looks like this:

```
default URI getContextURI() {
    return getURI();
}
```

Reply:
Fixed